### ISSUE TITLE
Is the documentation wrong?

### ISSUE THREAD

**Issue by taji** [Automattic/expect.js#113]
For using expect.js in the browser it says 

![image](https://cloud.githubusercontent.com/assets/702031/3712825/a90d049e-1538-11e4-8475-0b4454ca4d70.png)

Instead, shouldn't that last line contain _script src=**"index.js"**_?

**Comment by jasonkarns** [comment]
Yep. It was changed here: https://github.com/Automattic/expect.js/commit/fc222ebb46958819dd46e13990956a0d00fdfa83

**Comment by jasonkarns** [comment]
PR #104 addresses this
